tests/comments.test.js

```diff
@@ -0,0 +1,271 @@
+const request = require("supertest");
+const initApp = require("../server.js");
+const mongoose = require("mongoose");
+const commentModel = require("../models/comments_model.js");
+const postModel = require("../models/posts_model.js");
+
+let app;
+beforeAll(async () => {
+  app = await initApp();
+});
```

Comment:
All fields are required: postId, sender, content (in createComment)
what happens when thre is errir (getCommentsBySender)
what happens when there is a server error? getCommentById
what happens   when there is not post  id - "Post ID is required" (getCommentsByPost)
server error in update
server error id delete